### ISSUE TITLE
I tried to add new features to bedwars1058, but there was a dependency loss issue during construction

### ISSUE THREAD

**Issue by Lsy-291** [andrei1058/BedWars1058#788]
I have downloaded the complete source code of this plugin locally, but when I compiled it using Intellij IDEA, there were many dependency loss issues. My region is China, and I have tried using VPN to download dependencies, but it was unsuccessful

The missing partial dependencies are as follows：
Could not find artifact com.andrei1058.bedwars:versionsupport_1_8_R3:pom:23.3 in simonsators-repo (https://simonsator.de/repo/)
Could not find artifact com.andrei1058.bedwars:versionsupport_1_12_R1:pom:23.3 in simonsators-repo (https://simonsator.de/repo/)
Could not find artifact com.andrei1058.bedwars:resetadapter-slime:pom:23.3 in simonsators-repo (https://simonsator.de/repo/)
Could not find artifact com.andrei1058.bedwars:versionsupport_v1_16_R3:pom:23.3 in simonsators-repo (https://simonsator.de/repo/)
Could not find artifact com.andrei1058.bedwars:versionsupport_v1_17_R1:pom:23.3 in simonsators-repo (https://simonsator.de/repo/)
Could not find artifact com.andrei1058.bedwars:versionsupport_v1_18_R2:pom:23.3 in simonsators-repo (https://simonsator.de/repo/)
Could not find artifact com.andrei1058.bedwars:versionsupport_v1_19_R2:pom:23.3 in simonsators-repo (https://simonsator.de/repo/)
Could not find artifact com.andrei1058.bedwars:versionsupport-common:pom:23.3 in simonsators-repo (https://simonsator.de/repo/)

Can you tell me how to solve these problems
And my English is not very good. These words are all translated through machine translation
I am not very familiar with Java yet. If there is any discrepancy between my words and actions, please point it out
Thank you.

**Comment by Lsy-291** [comment]
I hope to solve some issues with Bedwars1058 and add interesting new features to it, but I am unable to compile it smoothly. Can someone help me

**Comment by andrei1058** [comment]
Repo.codemc.io

**Comment by Lsy-291** [comment]
> Repo.codemc.io

Oh, there is a link that I have checked and found to have some things I need. However, during the compilation of this project, there were many dependencies missing from other modules one after another. Perhaps my method of using this link was incorrect. Can you explain in detail how I should use this link? Alternatively, could you write a tutorial for compiling this project, which may be a bit excessive, or could you just provide me with a slightly more detailed explanation of how I should address these dependency issues? If possible, I would greatly appreciate it

**Comment by FallenCrystal** [comment]
先使用maven构建一遍. `mvn install`

**Comment by Lsy-291** [comment]
Yes, well done! I tried to delete all the environments, then downloaded Maven and the project source code again, and compiled using this code. Although there were still many errors in it, it was relatively easy to solve compared to relying on missing files to download. Your answer was very straightforward, and I successfully compiled it. Thank you very much and all those who participated in this issue!
